src/components/Hero/BenchmarkGraphs.jsx

```diff
@@ -17,7 +17,7 @@ const Wrapper = styled.div`
   }
 `;
 
-const mapData = (data, key) => data.map(item => ({ Percentage: (`${item.percentile * 100}%`), Value: Number(item[key].toFixed(2)) }));
+const mapData = (data, key) => data.map(item => ({ Percentage: (`${item.percentile * 100}%`), Value: typeof item[key] === 'number' && Number(item[key].toFixed(2)) }));
```

Comment:
what's the additional check for?  Are any of them not numbers?

Reply:
Without this the page will keep crashing for a while after we add new benchmarks